src/lang/en.json

```
@@ -880,5 +880,10 @@
     "useRemoteBrowser": "Use a Remote Browser",
     "deleteRemoteBrowserMessage": "Are you sure want to delete this Remote Browser for all monitors?",
     "GrafanaOncallUrl": "Grafana Oncall URL",
-    "Browser Screenshot": "Browser Screenshot"
+    "Browser Screenshot": "Browser Screenshot",
+    "wayToGetSevenIOApiKey": "Visit the dashboard under app.seven.io > developer > api key > the green add button",
+    "senderSevenIO": "Sending number or name",
+    "receiverSevenIO": "Receiving number",
+    "receiverInfoSevenIO": "If receiving number is not located in germany you have to add the country code in front of the number, eg for the country code 1 from the us: 117612121212 (instead of 017612121212)",
```

On second though you are correct, keeping this here might be better for translatablity.
What do you think about either one of these minor rewordings?

```suggestion
    "receiverInfoSevenIO": "If the receiving number is not located in Germany, you have to add the country code in front of the number. For example for the country code 1 from the US use 117612121212 instead of 017612121212",
```


```suggestion
    "receiverInfoSevenIO": "If the receiving number is not located in Germany, you have to add the country code in front of the number (e.g. for the country code 1 from the US use 117612121212 instead of 017612121212)",
```